### PR TITLE
Allow cron to always mail its output.

### PR DESCRIPTION
**Pull request by Heidistein**
I was used to the fact that redhat always mails its output. I enabled
the managed_cron bool, and added the --verbose startup argument (for
debugging) and all mail went away...

### CODE REVIEW

**Comment by LongLiveCHIEF** [comment]
The current errors you're seeing are a result of a bug with our [travis config](https://github.com/voxpupuli/modulesync_config/pull/483). It will be fixed as soon as we update modulesync. Once that is merged, you can rebase to re-run CI.

File: manifests/init.pp [hunk 1]
```diff
@@ -22,7 +22,8 @@
   Stdlib::Filemode $rules_configdir_mode = $logrotate::params::rules_configdir_mode,
   String $root_user                      = $logrotate::params::root_user,
   String $root_group                     = $logrotate::params::root_group,
-  Array[String] $logrotate_args          = []
+  Array[String] $logrotate_args          = [],
```
Comment (bastelfreak) [inline manifests/init.pp]: Can you please enforce a minimal string length like this:
```puppet
Array[String[1]]
```

**Comment by bastelfreak** [comment]
Hi @Heidistein. thanks for the PR. Can you take a look at the inline comment I made?

File: manifests/init.pp (hunk 1 above)
Comment (Heidistein) [reply manifests/init.pp]: Yes, I agree that is better. However, it belongs to pull request 80. I will fix it there.
https://github.com/Heidistein/puppet-logrotate/tree/pr80fix

Comment (Heidistein) [reply manifests/init.pp]: Fixed in PR #127 
I added only a training comma here, this was unrelated to this change.

**Comment by bastelfreak** [comment]
Hi, can you please rebase?

**Comment by Heidistein** [comment]
Rebase  done.

File: spec/defines/cron_spec.rb [hunk 2]
```diff
@@ -78,6 +78,18 @@
             with_content(%r{(\/usr\/local\/sbin\/logrotate -s \/var\/lib\/logrotate\/logrotate.status -m \/usr\/sbin\/mailer \/usr\/local\/etc\/logrotate.conf 2>&1)})
         }
       end
+
+      context 'With additional arguments' do
+        let(:pre_condition) { 'class {"::logrotate": cron_always_output => true}' }
```
Comment (bastelfreak) [inline spec/defines/cron_spec.rb]: FYI the `::` aren't needed in front of logrotate

**Comment by bastelfreak** [comment]
Thanks!

**Pull request merged**